src/Schema/Events/Consignments.ts

```diff
@@ -26,3 +26,113 @@ export interface ConsignmentSubmitted {
   user_email: string
   user_id?: string
 }
+
+/**
+ * First step of the consignment submission flow; user confirms details of their artwork.
+ *
+ * This schema describes events sent to Segment from [[artworkDetailsCompleted]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "artworkDetailsCompleted",
+ *    submission_id: "52521",
+ *    user_email: "kieranmbh@gmail.com"
+ *    user_id: "5cd6b173746be6109c86321d"
+ *  }
+ * ```
+ */
+export interface ArtworkDetailsCompleted {
+  action: ActionType.artworkDetailsCompleted
+  submission_id: string
+  user_email: string
+  user_id?: string
+}
+
+/**
+ * Second step of the consignment submission flow; user uploads photos of their artwork.
+ *
+ * This schema describes events sent to Segment from [[uploadPhotosCompleted]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "uploadPhotosCompleted",
+ *    submission_id: "52521",
+ *    user_email: "kieranmbh@gmail.com"
+ *    user_id: "5cd6b173746be6109c86321d"
+ *  }
+ * ```
+ */
+export interface UploadPhotosCompleted {
+  action: ActionType.uploadPhotosCompleted
+  submission_id: string
+  user_email: string
+  user_id?: string
+}
+
+/**
+ * Third step of the consignment submission flow; user adds contact information for their artwork.
+ *
+ * This schema describes events sent to Segment from [[contactInformationCompleted]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "contactInformationCompleted",
+ *    submission_id: "52521",
+ *    user_email: "kieranmbh@gmail.com"
+ *    user_id: "5cd6b173746be6109c86321d"
+ *  }
+ * ```
+ */
+export interface ContactInformationCompleted {
+  action: ActionType.contactInformationCompleted
+  submission_id: string
+  user_email: string
+  user_id?: string
+}
+
+/**
+ * User clicks/taps to re-enter the submission flow after successfully submitting an artwork for consignment.
+ *
+ * This schema describes events sent to Segment from [[submitAnotherArtwork]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "submitAnotherArtwork",
+ *    submission_id: "52521",
+ *    user_email: "kieranmbh@gmail.com"
+ *    user_id: "5cd6b173746be6109c86321d"
+ *  }
+ * ```
+ */
+export interface SubmitAnotherArtwork {
+  action: ActionType.submitAnotherArtwork
+  submission_id: string
+  user_email: string
+  user_id?: string
+}
+
+/**
+ * User clicks/taps to view the artwork they just submitted in MyCollection.
+ *
+ * This schema describes events sent to Segment from [[viewArtworkMyCollection]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "viewArtworkMyCollection",
+ *    submission_id: "52521",
+ *    user_email: "kieranmbh@gmail.com"
+ *    user_id: "5cd6b173746be6109c86321d"
+ *  }
+ * ```
+ */
+export interface ViewArtworkMyCollection {
+  action: ActionType.viewArtworkMyCollection
```

Comment:
let's also add a context_owner_type: `consignmentSubmissionThankYou` (in addition to the context_module)